packages/ui/src/ui/components/Dialog/controls/RoleListControl/RoleListControl.tsx

```diff
@@ -184,20 +176,22 @@ export function roleListValueToSubjectList(value: Props['value']) {
     ];
 }
 
-function manyListDataItemToSubjectList(manyListDataItem: EditableManyListsItemType<RoleConverted>) {
+function manyListDataItemToSubjectList(
+    manyListDataItem: EditableManyListsItemType<RoleConverted>,
+): Array<ResponsibleType> {
     const {data} = manyListDataItem || {};
     return _.map(
         _.filter(data, ({removed}) => !removed),
         ({data}) => {
             const {type, value} = data || {};
-            return {type, value};
+            return {type: type!, value: value!};
```

Comment:
Judging by the code above, the variables may be "undefined". Are you sure you should put "!"?